Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py

```diff
@@ -191,11 +190,11 @@ def PtypString8(data_value):
     def PtypString(data_value):
         if data_value:
             try:
+                data_value = data_value.decode("utf-16-le", errors="ignore").replace('\x00', '')
+            except UnicodeDecodeError:
```

Comment:
is there any case where a `UnicodeDecodeError` error will be raised? please comment inline the code to further explain.